unittest/lstm_test.h

```diff
@@ -119,9 +119,9 @@ class LSTMTrainerTest : public testing::Test {
       trainer_->MaintainCheckpoints(nullptr, &log_str);
       iteration = trainer_->training_iteration();
       mean_error *= 100.0 / kBatchIterations;
-      LOG(INFO) << log_str.c_str();
-      LOG(INFO) << "Best error = " << best_error << "\n" ;
-      LOG(INFO) << "Mean error = " << mean_error << "\n" ;
+      //LOG(INFO) << log_str.c_str();
```

Comment:
Why do you want to keep those statements as a comment instead of removing them completely?

Reply:
You are right. They can be removed. I will modify the PR.

Reply:
Maybe make them as LOG(DEBUG) or LOG(TRACE)?

Reply:
Removed.